modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -13,6 +13,263 @@
+
+[discrete]
+==== Option to Hide Unwanted Shares:
+
+We understand that sometimes you may receive shares that are not relevant or wanted. To address this, we have introduced a feature that allows you to hide such shares. By hiding a share, you can maintain a clear view of the shares that are important to you, ensuring your "Shared with Me" section remains organized and clutter-free.
+
+**Access to Hidden Shares:**
+If you change your mind and wish to view a hidden share, you can easily do so. A dedicated area for hidden shares has been added, allowing you to revisit and manage any shares you have previously hidden.
+
+[discrete]
+==== Enhanced Search Filter:
+
+To help you quickly find specific shares, we have improved the search functionality within the "Shared with Me" section. This enhanced search filter enables you to efficiently locate files based on various criteria.
+
+[discrete]
+==== Sync Feature for Better Control:
+
+Gain control over which shared files are available for sync on your mobile and desktop devices. This feature is especially useful for managing large files. You can now choose to exclude certain files from syncing right from the start.
+
+[discrete]
+=== Shortcuts
+
+This new feature is designed to improve navigation and access within the platform, making your experience more efficient and integrated.
+
+**Link to External: Webpage:**
+
+You can now create shortcuts that link directly to external websites. This feature allows for quick access to frequently used online resources, right from within ownCloud Infinite Scale.
+
+**Internal Linking to File, Folder or a Space**
+
+The Shortcut feature also allows you to create links to internal files, folders, or Spaces. This improves organization and accessibility to important documents and areas within ownCloud, which is especially useful in collaborative scenarios.
+
+[discrete]
+=== Improved Tags
+
+You can now add tags to their files directly from the file's details panel. This update eliminates the need to navigate away from the details panel, streamlining the process of organizing and categorizing your documents. With tagging now integrated into the details panel, the process becomes more intuitive and user-friendly.
+
+[discrete]
+=== New action: Duplicate a Space
+
+This action is designed to enhance the flexibility and efficiency of space management for our users.
+You have now the option to create a copy of an existing Space. This duplication includes all files and folder structures within the Space. The duplicated Space will be free of any existing members, shares, sharing links or tags. This precaution is taken to prevent accidental data leakage and to ensure that the Space Manager can start from scratch, setting up a new Space for team collaboration as needed.
```

Comment:
```suggestion
You now have the option to create a copy of an existing Space. This duplication includes all files and folder structures within the Space. The duplicated Space will be free of any existing members, shares, sharing links or tags. This precaution is taken to prevent accidental data leakage and to ensure that the Space Manager can start from scratch, setting up a new Space for team collaboration as needed.
```